### PR TITLE
change to using flutter run --resident

### PR DESCRIPTION
**Pull request by devoncarew**
-- change to using `flutter run --resident`; it's a long-running flutter launch, designed to speed up the edit/refresh dev. cycle

@danrubel

### CODE REVIEW

File: lib/flutter/flutter_launch.dart [hunk 1]
```diff
@@ -148,78 +156,54 @@ class _LaunchInstance {
...
-      if (_device != null) {
-        args.add('--device-id');
-        args.add(_device.id);
-      }
-
-      ProcessRunner flutterStop = _flutter(flutter, args, cwd: project.path);
-      flutterStop.execSimple().catchError((e) {
-        _logger.finer('Error from flutter stop', e);
+      return new Future.delayed(new Duration(milliseconds: 250), () {
+        return _runner.kill();
```
Comment (danrubel) [inline lib/flutter/flutter_launch.dart]: Will _runner ever be null here? Is there a race condition? Or is _runner never set to null once it is assigned?

**Comment by danrubel** [comment]
LGTM

File: lib/flutter/flutter_launch.dart (hunk 1 above)
Comment (devoncarew) [inline lib/flutter/flutter_launch.dart]: I'll add a `?` operator and cleanup up the `_runner` lifecycle a bit.

**Comment by devoncarew** [comment]
Having a bit of trouble with this locally - it's not killing the app on the device when the user hits stop. Will do some triaging -

**Pull request merged**